.run/Fix PHP-CS-Fixer.run.xml

```diff
@@ -3,4 +3,4 @@
     <CommandLine workingDirectory="$PROJECT_DIR$" />
     <method v="2" />
   </configuration>
-</component>
```

Comment:
It would be nice to revert these changes because `.run.xml` files are managed by the IDE and they are generated without trailing empty line.